packages/apollo-server-core/src/plugin/landingPage/default/types.ts

```diff
@@ -75,7 +75,7 @@ export interface ApolloServerPluginEmbeddedLandingPageProductionDefaultOptions
    * relying on the server's ApolloConfig, because if your server is publicly
    * accessible you may not want to display the graph ref publicly.)
    */
-  graphRef: string;
+  graphRef?: string;
   /**
    * Users can configure their landing page to render an embedded Explorer if
    * given a graphRef, or an embedded Sandbox if there is not graphRef provided.
```

Comment:
OK, so the change above matches this comment. Got it. But one thing I'm confused about: is ApolloServerPluginEmbeddedLandingPageProductionConfigOptions specific to Explorer or does it configure sandbox too?  Looking at the code it seems like it is explorer specific.  Maybe we should rename ApolloServerPluginEmbeddedLandingPageProductionConfigOptions to ApolloServerPluginLandingPageEmbeddedExplorerOptions, and do even more TS madness to only allow you to pass it with a graph ref.

(BTW in any case I think the type name probably doesn't need to have both "Config" and "Options" in it.  Fortunately this type isn't exported so changing it isn't a compat issue.  Actually given that it's not exported it could probably have a shorter name like EmbeddedExplorerOptions.)

Additionally, this change seems to require a docs update, since the production landing page doesn't appear to mention Sandbox at all?

Reply:
I think the graphRef should be required if you pass `embed: true` in  a production landing page. The comment here is wrong, I don't actually think we want to show the sandbox on production endpoints ever. Fixing it now